packages/fxa-settings/src/components/Settings/UnitRowTwoStepAuth/index.tsx

```diff
@@ -155,17 +150,24 @@ export const UnitRowTwoStepAuth = ({
           onCtaClick={() => {
             navigate(replaceCodesRoute);
           }}
+          key={1}
         />
       );
-    }
-    if (backupPhoneSubRowProps?.onCtaClick) {
-      subRows.push(
-        <BackupPhoneSubRow
-          onCtaClick={backupPhoneSubRowProps.onCtaClick}
-          onDeleteClick={backupPhoneSubRowProps.onDeleteClick}
-          phoneNumber={backupPhoneSubRowProps.phoneNumber}
-        />
-      );
+      if (
+        config.featureFlags?.enableAdding2FABackupPhone === true &&
+        (recoveryPhone.available === true || recoveryPhone.exists === true)
+      ) {
+        subRows.push(
+          <BackupPhoneSubRow
+            onCtaClick={() => {
+              navigate(`${SETTINGS_PATH}/recovery_phone/setup`);
+            }}
+            onDeleteClick={() => {}}
```

Comment:
Not sure if should be added here or in @vbudhram functional tests ticket
```suggestion
            onDeleteClick={() => {navigate(`${SETTINGS_PATH}/recovery_phone/remove`)}}
```

Reply:
Heh I figured this would've been added in the delete ticket, but I can add after rebasing against `main` which I'm about to do now.

Reply:
Added.